README.md

```diff
@@ -110,51 +159,51 @@ will go to the tscircuit registry instead of the npm registry.
 @tsci:registry=https://registry-api.tscircuit.com/npm
 ```
 
-### Can I use a different registry?
+### 5. Can I use a different registry?
 
 Yes! You can use the same trick and modify the `.npmrc` file for your project. You can run your own npm registry with [verdaccio](https://verdaccio.org/)
 
-### Why are some things upside down?
+### 6. Why are some things upside down?
 
 If you see something upside down, it's because mid-April the Y Axis was flipped so that "up" would mean the Y value would increase for both schematics and PCBs. In web development, the
 Y axis is negative as you go up, but [this probably isn't a good choice an engineering tool](https://x.com/seveibar/status/1780992701800034726) so now the Y Axis is positive and some
 things are upside down until they're fixed.
 
-### Can I test this in my browser?
+### 7. Can I test this in my browser?
 
 Yes! There is a [playground tool!](https://tscircuit.com/playground)
 
-### Do I have to specify the position of every component?
+### 8. Do I have to specify the position of every component?
 
 I'm working on autolayout functionality where you will hopefully only need to
 tweak things like the spacing. The goal of anything auto
 
-### Is the auto-routing good?
+### 9. Is the auto-routing good?
 
 The auto-routing is very basic. You can adjust the paths by adding `pcb_route_hints`
 to a `<trace />`. Over time, we'll get good, fast auto-routing algorithms and also
 ways to "bake & cache" a route over a longer time period.
 
-### Can I export into other tools for routing?
+### 10. Can I export into other tools for routing?
 
 This isn't currently possible but a major current objective. If you're interested in writing a module that can convert [tscircuit soup JSON](https://docs.tscircuit.com/api-reference/advanced/soup)
 into another editor's format, reach out on our [campfire](https://tscircuit.com/community/join-redirect)!!
 
-### I found a bug or have an idea for a feature, what should I do?
+### 11. I found a bug or have an idea for a feature, what should I do?
 
 Please [create an issue](https://github.com/tscircuit/tscircuit/issues)!
 
-### How can I follow along?
+### 11. How can I follow along?
 
 - [@seveibar](https://x.com/seveibar) is the main author, he tweets about tscircuit and does [development livestreams](https://www.twitch.tv/seveibar)
 - [@tscircuit](https://x.com/tscircuit) for major tscircuit releases, features and discussions
 - [tscircuit discord](https://discord.gg/6X3PYhtj) and [tscircuit campfire](https://tscircuit.com/community/join-redirect)
 
-### Does this use AI?
+### 12. Does this use AI?
 
 No, but AI is pretty good at generating circuits! Try it with copilot!
 
-### What big features are coming?
+### 13. What big features are coming?
```

Comment:
why number these?

Reply:
I think having numbers along with the question helps with better readability in FAQs. 🤔

Reply:
AFAIK they can be used for 3 things:

- Indicating a semantic order
- Referencing
- Frequency/Popularity

I think in this case none of these are appicable